docs/development-environment.md

```diff
@@ -0,0 +1,174 @@
+
+Then start the services:
+
+```shell
+$ cd udata
+$ docker-compose up
+```
+
+On the very first run it will download and install Docker images which takes a while depending of your connection.
+
+!!! warning
+    Test your _docker-compose_ is running successfully entering `curl http://localhost:9200`.
+    It should output a JSON search response.
+    If you have no output at all for too long,
+    check the [IPv6 possible issue](https://github.com/docker/docker/issues/2174#issuecomment-35697655).
+
+## Python and virtual environment
+
+It is recommended to work within a virtualenv to ensure proper dependencies isolation.
+If you're not familiar with that concept, read [Python Virtual Environments - a Primer][].
+
+Alright, now you can [install virtualenv][install-virtualenv] and then type these commands knowing what you are doing:
+
+```shell
+$ virtualenv --python=python2.7 venv
+$ source venv/bin/activate
+$ pip install -r requirements/develop.pip
+$ pip install -e .
+```
+
+Some dependencies have an optionnal compilation support for Cython
+resulting in more performances (mostly XML harvesting).
+To enable it, you need to install Cython before all other dependencies:
+
+```shell
+$ pip install Cython
+$ pip install -r requirements/develop.pip
+$ pip install -e .
+```
+
+## NodeJS and modules
+
+You need NodeJS 6.2.4 (but your can check the version used in the `.nvmrc` file at the repository root)
+If it's not already installed or you have a different version,
+you should consider [installing NVM][nvm-install] which uses the existing `.nvmrc`.
+
+```shell
+$ nvm install
+$ nvm use
+```
+
+Then install JavaScript dependencies:
+
+```shell
+$ npm install
+```
+
+Once it's done, you should be able to run the build command for JS and CSS:
+
+```shell
+$ inv assets_build
+```
+
+!!! note
+If you plan to hack on statics (JS, CSS files), a dedicated command `inv assets_watch` will watch these files and recompile (the modified part only!) on each save.
+
+
+## Running the project
+
+As long as you have the middlewares up, you can use `invoke` to launch the development services
+(you might want to have each one runnning in a terminal):
+
+```shell
+$ inv serve  # Start the development server
+$ inv worker  # Start a worker process
+$ inv beat  # Start a scheduler process
+$ inv assets_watch  # Continously watch and build assets
+```
+
+When you have the development server running,
+you can open your browser to <http://localhost:7000>.
+Everything is up and running!
+
+!!! error
+    If not, it's time for your first contribution to improve the documentation!
+    But first let's try to figure out together what went wrong on [Gitter][].
+
+You need to initialize some data before pushing uData to it's full potential:
+
+```shell
+# Initialize database, indexes...
+$ udata init
+# Optionnaly fetch and load some licenses from another udata instance
+$ udata licenses https://www.data.gouv.fr/api/1/datasets/licenses
```

Comment:
Not anymore, this why I added `Optionnaly`. I kept the line because the opendefinitions list does not include the french licenses.